src/bin/rav1e.rs

```diff
@@ -139,17 +145,17 @@ fn process_frame<T: Pixel, D: Decoder>(
            .expect("Unable to write to two-pass data file.");
         }
       }
-      return None;
+      return Ok(None);
     }
-    Err(EncoderStatus::Failure) => {
-      panic!("Failed to encode video");
+    e @ Err(EncoderStatus::Failure) => {
+      let _ = e.map_err(|e| e.context("Failed to encode video"))?;
     }
-    Err(EncoderStatus::NotReady) => {
-      panic!("Mis-managed handling of two-pass stats data");
+    e @ Err(EncoderStatus::NotReady) => {
+      let _ = e.map_err(|e| e.context("Mis-managed handling of two-pass stats data"))?;
```

Comment:
No need for the hyphen in "mismanaged".

Reply:
I can conflate the fix in this patch if you want.

Reply:
Done.